src/openrpc/lifecycle.json

```diff
@@ -1,379 +1,390 @@
 {
-  "openrpc": "1.2.4",
-  "info": {
-    "title": "Lifecycle",
-    "description": "Methods and events for responding to lifecycle changes in your app",
-    "version": "0.0.0"
-  },
-  "methods": [
-    {
-      "name": "ready",
-      "tags": [
-        {
-          "name": "calls-metrics"
-        },
-        {
-          "name": "capabilities",
-          "x-uses": ["xrn:firebolt:capability:lifecycle:ready"]
-        },
-        {
-          "name": "exclude-from-sdk"
-        }
-      ],
-      "summary": "Notify the platform that the app is ready",
-      "params": [],
-      "result": {
-        "name": "result",
-        "schema": {
-          "const": null
-        }
-      },
-      "examples": [
-        {
-          "name": "Let the platform know that your app is ready",
-          "params": [],
-          "result": {
-            "name": "Default Result",
-            "value": null
-          }
-        }
-      ]
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "Lifecycle",
+        "description": "Methods and events for responding to lifecycle changes in your app",
+        "version": "0.0.0"
     },
-    {
-      "name": "close",
-      "tags": [
-        {
-          "name": "capabilities",
-          "x-uses": ["xrn:firebolt:capability:lifecycle:state"]
-        }
-      ],
-      "summary": "Request that the platform move your app out of focus",
-      "params": [
-        {
-          "name": "reason",
-          "summary": "The reason the app is requesting to be closed",
-          "required": true,
-          "schema": {
-            "$ref": "https://meta.rdkcentral.com/firebolt/schemas/lifecycle#/definitions/CloseReason"
-          }
-        }
-      ],
-      "result": {
-        "name": "success",
-        "schema": {
-          "const": null
-        }
-      },
-      "examples": [
-        {
-          "name": "Close the app when the user presses back on the app home screen",
-          "params": [
-            {
-              "name": "reason",
-              "value": "remoteButton"
-            }
-          ],
-          "result": {
-            "name": "Default Result",
-            "value": null
-          }
+    "methods": [
+        {
+            "name": "ready",
+            "tags": [
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:lifecycle:ready"
+                    ]
+                },
+                {
+                    "name": "exclude-from-sdk"
```

Comment:
We need this to be in the C++ SDK, so I don't think this tag is correct, unless we want to say C++ includes all APIs even with this tag.

I was imagining something more like a list of capabilities to exclude from each language, somewhere inside the sdk.config.json, e.g.:

```json
{
  "languages": {
    "javascript": {
      "excludeModules": [
        "Lifecycle"
      ]
    }
  }
}
```

Reply:
Yea @kevinshahfws let's sync up on how resolve this and come up with a quick solution.

We'll probably need more granular, API-specific configs (e.g. the W3C media caps API should be excluded from the JS SDK, but the other MC APIs should still be there).